core/src/test/scala/org/apache/spark/SerializerHelper.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+
+trait SerializerHelper {
```

Comment:
I'd consider naming this something like `SerializerTestUtils` to avoid confusion with the existing non-test SerializerHelper defined at https://github.com/apache/spark/blob/bbb8aca0b51008bf65ba8f9232ba96c166e84f8e/core/src/main/scala/org/apache/spark/serializer/SerializerHelper.scala#L27

Reply:
sounds good!